checkov/terraform/graph_builder/local_graph.py

```diff
@@ -244,12 +249,15 @@ def _build_edges_for_vertex(self, origin_node_index: int, vertex: TerraformBlock
                             referenced_module_object = module.get("source_module_object")
                             if not self.use_new_tf_parser:
                                 source_module_object = referenced_module_object if source_module_object else None
-                            dest_node_index = self._find_vertex_index_relative_to_path(
-                                vertex_reference.block_type, reference_name, referenced_module_path,
-                                vertex.module_dependency,
-                                vertex.module_dependency_num, referenced_module_idx,
-                                source_module_object=source_module_object
-                            )
+                            if referenced_module_path is None:
+                                dest_node_index = -1
```

Comment:
is this correct to use? why not just returning here?

Reply:
since it is inside a for loop it can change with the next iteration, that's why I can't just return it.